app/interfaces/interfaces.ts

```diff
@@ -3,29 +3,24 @@ export interface Coords {
 	latitude: number;
 }
 
-
-export interface Forecast {
-	message: string;
-	cod: string;
-	count: number;
-	list: cityDetails[];
-}
-
-interface cityDetails {
+export interface cityDetails extends weatherData {
```

Comment:
It's actualy not very good case for using extending. 
To keep CityDeatails simple you could create one more intreface, let's say MainDetails 

interface MainDetails {
                temp: number;
  		humidity: number;		  		
  		pressure: number;		  		
  		temp_max: number;		  		
  		temp_min: number;
}

and then use it in CityDetails 
interface CityDetails {
....
main: MainDetails
....
 }